config/config.go

```diff
@@ -167,6 +167,7 @@ type SlaveOptionsConfig struct {
 	CallTimeout                     int    `json:"call_timeout"`
 	PingTimeout                     int    `json:"ping_timeout"`
 	RPCPoolSize                     int    `json:"rpc_pool_size"`
+	SecondsToCheckKeySpaceChanges   int    `json:"seconds_to_check_key_space_changes"`
```

Comment:
Pls name it `keyspace_sync_interval`

Reply:
Done! currently there is a test that doesn't pass but is not related to this ticket. (Hostchecker test)